gcloud-java-translate/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>gcloud-java-translate</artifactId>
+  <packaging>jar</packaging>
+  <name>GCloud Java translate</name>
+  <url>https://github.com/GoogleCloudPlatform/gcloud-java/tree/master/gcloud-java-translate</url>
+  <description>
+    Java idiomatic client for Google Translate.
+  </description>
+  <parent>
```

Comment:
Any chance you could put more whitespace in here to make it a bit more readable?  ideally between major nodes.  Ok to say no, as this isn't example code, but folks do look at our code as how to write well.